portfolio/src/main/webapp/main_page_clickables/aboutme.html

```diff
@@ -0,0 +1,20 @@
+<p>
+    &emsp; Ever since I was a kid, what has always interested me the most is solving problems. 
```

Comment:
Using `&emsp;` here looks really nice!

I think you can also control indentation with CSS, see this for an example: https://jsfiddle.net/jxoyqh89/1/